docs/src/pages/components/dividers/ListDividers.js

```diff
@@ -1,23 +1,20 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Divider from '@material-ui/core/Divider';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: '100%',
-    maxWidth: 360,
-    backgroundColor: theme.palette.background.paper,
-  },
-}));
-
 export default function ListDividers() {
-  const classes = useStyles();
-
   return (
-    <List component="nav" className={classes.root} aria-label="mailbox folders">
+    <List
```

Comment:
this change is not flying, we lost the inline preview:

- HEAD: https://next.material-ui.com/components/dividers/#list-dividers
- PR: https://deploy-preview-25145--material-ui.netlify.app/components/dividers/#list-dividers

Reply:
@oliviertassinari Can't solve this. Try injecting the css in various ways with no success.

The same goes for the Vertical divider demo